packages/analytics-browser/src/browser-client.ts

```diff
@@ -45,9 +47,22 @@ export class AmplitudeBrowser extends AmplitudeCore<BrowserConfig> {
       isNewSession = true;
     }
 
+    // Set up the analytics connector to integrate with the experiment SDK.
+    // Send events from the experiment SDK and forward identifies to the
+    // identity store.
+    const connector = getAnalyticsConnector();
+    connector.eventBridge.setEventReceiver((event) => {
+      void this.track(event.eventType, event.eventProperties);
+    });
+    connector.identityStore.setIdentity({
+      userId: this.config.userId,
+      deviceId: this.config.deviceId,
+    });
+
```

Comment:
Hey @bgiori , is analytics connector optional or required for analytics browser? For the customers who do not use experiment SDK, do they need this analytics connector? cc: @kevinpagtakhan

Reply:
with these changes, it will emit and listen for 'messages' regardless but experiment SDK may/may not be on the other end.

Reply:
Correct me if I am wrong. "@amplitude/analytics-connector" will not be tree shaking because the source file "./utils/analytics-connector" imports it. The TS browser bundle might include the "@amplitude/analytics-connector".

Reply:
Correct, not treeshaken.

Reply:
Not treeshaken. I am not sure what the requirements are for treeshaking this. If you have suggestions about how to make this tree-shakable, I'm happy to implement.

Reply:
I think you can move `getAnalyticsConnector` and `IdentityEventSender` into a separate file and then try the following 
```js
if (this.config.enableExperiment) {
  const { getAnalyticsConnector, IdentityEventSender } = await import('FILE_NAME');
  const connector = getAnalyticsConnector();
  connector.eventBridge.setEventReceiver((event) => {
    void this.track(event.eventType, event.eventProperties);
  });
  connector.identityStore.setIdentity({
    userId: this.config.userId,
    deviceId: this.config.deviceId,
  });

  await this.add(new IdentityEventSender());
}
```

Does experiment SDK have timeout or need to initialize after core SDK initialization? With this change, core SDK might take longer time to complete the initialization.

Reply:
I want to avoid additional configuration on the analytics SDK, might be worth the tradeoff since the package is pretty small. 

We can always re-optimize later since this is all internal.